app/src/main/java/com/randomappsinc/studentpicker/home/HomeActivity.java

```diff
@@ -28,13 +28,15 @@
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 
+import static com.randomappsinc.studentpicker.importdata.ImportFromFileActivity.IS_TEXT_FILE;
+
 public class HomeActivity extends StandardActivity implements
         BottomNavigationView.Listener, CreateListDialog.Listener, PaymentManager.Listener {
 
     private static final int NUM_APP_OPENS_FOR_RATING_ASK = 5;
 
     private static final int IMPORT_FILE_REQUEST_CODE = 1;
```

Comment:
Let's make the old code more readable and able to support our change.
```suggestion
    private static final int IMPORT_TXT_REQUEST_CODE = 1;
```